Assets/Scripts/Dialogue scripts/Dialogue.cs

```diff
@@ -1,197 +1,184 @@
-﻿using UnityEngine;
-using UnityEngine.UI;
-using System.Collections.Generic;
+﻿using System;
 using System.Collections;
 using System.Text;
-using System;
+using UnityEngine;
 
-[RequireComponent(typeof(BoxCollider2D))]
-public class Dialogue : Interacted
+namespace Dialogue_scripts
 {
-    [TextArea] public string[] _textBox; // string array for inputting separate dialogue boxes
-
-    [TextArea] public string[] _hiddenTextBox;
-
-    [SerializeField] private RenderDialogue _pageRender;
-
-    [SerializeField] GameObject rayCast;
-
-    [SerializeField] GameObject dialogueBox;
-
-    public JournalInventoryScript joInvScript;
-
-    [SerializeField] GameObject accessItem;
-
-    [SerializeField] GameObject hiddenItem;
-
-    [SerializeField] GameObject item;
-
-    [SerializeField] private DeathMovement _movement;
-
-    public int _itemClass;
-
-    public int _pageClass;
-
-    BoxCollider2D _myCollider;
-
-    SpriteRenderer _myRenderer;
-
-    public bool isIntro = false;
-
-
-    private void Start()
+    [RequireComponent(typeof(BoxCollider2D))]
+    public class Dialogue : Interacted
     {
-        dialogueBox.SetActive(false);
+        [TextArea] [SerializeField] private string[] _textBox; // string array for inputting separate dialogue boxes
+        [TextArea] [SerializeField] private string[] _hiddenTextBox;
+        [SerializeField] private RenderDialogue _pageRender;
+        [SerializeField] private GameObject _rayCast;
+        [SerializeField] private GameObject _dialogueBox;
+        [SerializeField] private JournalInventoryScript _joInvScript;
+        [SerializeField] private GameObject _accessItem;
+        [SerializeField] private GameObject _hiddenItem;
+        [SerializeField] private GameObject _item;
+        [SerializeField] private DeathMovement _movement;
+        [SerializeField] private int _itemClass;
+        [SerializeField] public int pageClass;
+        private BoxCollider2D _myCollider;
+        private SpriteRenderer _myRenderer;
+        [SerializeField] private bool isIntro;
+
+
+        private void Start()
+        {
+            _dialogueBox.SetActive(false);
 
-        _myCollider = GetComponent<BoxCollider2D>();
+            _myCollider = GetComponent<BoxCollider2D>();
 
-        _myRenderer = gameObject.GetComponent<SpriteRenderer>();
+            _myRenderer = gameObject.GetComponent<SpriteRenderer>();
```

Comment:
This is brittle. There is no `RequireComponent` ensuring it exists and makes a silent assumption it'll simply be there. either `RequireComponent` it or get it via inspector.